e2e/assertions/compareScreenshot.js

```diff
@@ -2,10 +2,16 @@
 const resemble = require('node-resemble-js')
 const fs = require('fs')
 const path = require('path')
+const readline = require('readline')
 
 const { tolerance } = require('../config')
 const { getVisualRegressionFolders } = require('../utils')
 
+const updateScreenshots = process.env.UPDATE_SCREENSHOTS
+const updateAllScreenshots = process.env.UPDATE_ALL_SCREENSHOTS
```

Comment:
You can simplify further by doing the `=== 'true'` comparison here.

```js
const updateScreenshots = process.env.UPDATE_SCREENSHOTS === 'true'
const updateAllScreenshots = process.env.UPDATE_ALL_SCREENSHOTS === 'true'
```

Reply:
Good idea!